### PR TITLE
Hotfix for failing linking to project search

### PR DESCRIPTION
**Pull request by ignapas**
Tackles https://www.wrike.com/open.htm?id=1408704796

### CODE REVIEW

**Review by egauzens — CHANGES_REQUESTED** [review]
Testing the changes locally, and this does not work. It still shows 11 results when navigating the the link /about/projects?consortiaType=SPARC&search=implantable&selectedProjectsAnatomicalFocusIds=Vagus

**Comment by ignapas** [comment]
> Testing the changes locally, and this does not work. It still shows 11 results when navigating the the link /about/projects?consortiaType=SPARC&search=implantable&selectedProjectsAnatomicalFocusIds=Vagus

I thought it was working, sorry... I made another change to avoid executing the `consortiaType` watcher

https://github.com/nih-sparc/sparc-app-2/assets/4764217/113ccd24-d167-4600-928d-69ec43a494c7

Note that I cannot load the home page...

File: components/FilesTable/FilesTable.vue [hunk 1]
```diff
@@ -680,7 +680,10 @@ export default {
       const fileType = scope.row.fileType.toLowerCase()
       const contentType = contentTypes[fileType]
 
-      const requestUrl = `${this.$config.public.portal_api}/download?s3BucketName=${s3BucketName}&key=${filePath}&contentType=${contentType}`
+      const requestUrl = new URL(this.$config.public.portal_api + '/download')
```
Comment (egauzens) [inline components/FilesTable/FilesTable.vue:683]: I think you can revert these changes, they do not appear related

File: pages/about/projects/index.vue [hunk 2]
```diff
@@ -281,35 +281,31 @@ export default {
...
         this.fetchResults()
-      },
-      immediate: true
+      }
     },
   },
 
   beforeMount: function () {
     this.windowWidth = window.innerWidth
+    this.fetchConsortiaStyle(this.consortiaType.id)
```
Comment (egauzens) [inline pages/about/projects/index.vue:308]: you cannot move this here. We need to fetch the consortia style when a user switches tabs. It is broken if this is moved to here since this is only ran on initial page load and page refresh

**Review by egauzens — CHANGES_REQUESTED** [review]
This breaks the consortia styling

File: pages/about/projects/index.vue (hunk 2 above)
Comment (ignapas) [reply pages/about/projects/index.vue:309]: sorry for that. it should be working properly now...

Comment (egauzens) [reply pages/about/projects/index.vue:309]: I don't see any changes....

**Review by egauzens — CHANGES_REQUESTED** [review]

File: pages/about/projects/index.vue (hunk 2 above)
Comment (egauzens) [reply pages/about/projects/index.vue:309]: The styling works properly again, but the number of results is still wrong

File: components/FilesTable/FilesTable.vue (hunk 1 above)
Comment (egauzens) [reply components/FilesTable/FilesTable.vue:683]: please revert these changes

**Review by egauzens — APPROVED** [review]
Works!
